src/Path.roc

```diff
@@ -256,7 +263,7 @@ WindowsRoot : []
 #                     Str.startsWithUtf8 pathStr prefixBytes
 #                 FromStr prefixStr ->
 #                     Str.startsWith pathStr prefixStr
-## Returns `Bool.true` if the first path ends with the second.
+## Returns [Bool.true](https://www.roc-lang.org/builtins/Bool#true) if the first path ends with the second.
```

Comment:
Does this work without the parens? e.g. just `[Bool.true]` on its own, and then the automatic documentation link generator takes care of the rest? (If not, we should probably make it work!)

Reply:
Yeah, these don't work currently as we don't generate docs for builtins alongside the platform docs.

Reply:
Reverted all builtin links back to [Bool.true] style. Raised as an idea in Zulip. This will likely be resolved through a PR to `roc-lang/roc` cli docs gen, so another PR will be required over there to resolve.